Makefile

```
@@ -114,9 +118,18 @@ uninstall:
 # TEST
 ###############################################################################
 
-.PHONY: test test-cl-quil test-quilc test-quilt test-quilec
+.PHONY: test test-cl-quil test-quilc test-quilt test-quilec get_latest_quilc_deps
 test: test-cl-quil test-quilt test-quilec test-quilc
 
+# When testing, you may wish to pull QVM and Magicl from appropriate
+# repo/refspec prior to test. NOTE: this has side effects on
+# RIGETTI_LISP_LIBRARY_HOME directory, namely cloning deps
+# directories. Generally, this is meant to be run in a VM, typically
+# Docker-supplied, with an initially empty build directory.
+get_latest_quilc_deps:
```

```suggestion
get-latest-quilc-deps:
```
(just for consistency with the other targets)